requirements.txt

```diff
@@ -1,2 +1,18 @@
 xarray==0.19.0
 -e git+https://github.com/OpenHydrology/lmoments3.git@7e19f97c23019ca68cbd526b8bd417c412438f1c#egg=lmoments3
+cartopy==0.20.1
+geopandas==0.10.2
+geoviews==1.9.2
+holoviews==1.14.6
+hvplot==0.7.3
+intake-xarray==0.5.0
+lmoments3
+matplotlib==3.4.3
+numpy==1.21.2
+pandas==1.3.4
+panel==0.12.4
```

Comment:
While we're updating these, I think there are additional dependencies to get the panel stuff to work: https://panel.holoviz.org/user_guide/Server_Deployment.html

Reply:
Only the minimal set of application dependencies should be specified here, essentially everything that's imported, and not the dependencies of dependencies unless there are particular versions to avoid that are known to break things. I'm not seeing any more import errors on my end due to missing packages when installing in a clean virtualenv. The panel server deployment only applies to Panel apps and dashboards, not notebook usage of it.

Reply:
Okay, we'll make a separate issue of why the panels are not working outside of the pangeo docker image.